index.html

```diff
@@ -2,6 +2,10 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+    <meta
+      name="description"
+      content="A personal portfolio website is a great way to showcase your work and your skills to potential employers."
+    />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>My Portfolio</title>
```

Comment:
- [ ] [Optional] I would suggest you add `Mark's portfolio or
Mark full-stack web developer` for better page titles. Kindly use your name in place of Mark.

Reply:
ooh ok, thanks for the great feedback 💯